### PR TITLE
Filipino Translation devise.fil.yml

### PR DESCRIPTION
**Pull request by vinzruzell**
# Context
I have translated devise.en.yml to devise.fil.yml (Filipino Words).

# Summary of Changes
Partial changes from Line 1 to 30.

# Checklist

- [ ] Tested Mobile Responsiveness
- [ ] Added Unit Tests
- [ ] CI Passes
- [ ] Deploys to Heroku on test Correctly (Maintainers will handle)
- [ ] Added Documentation (Service and Code when required)

# Screenshots

## After
# Context
I have translated devise.en.yml to devise.fil.yml (Filipino Words).

# Summary of Changes
Completely translated 60 lines of devise.en.yml into Filipino language.

# Checklist

- [ ] Tested Mobile Responsiveness
- [ ] Added Unit Tests
- [ ] CI Passes
- [ ] Deploys to Heroku on test Correctly (Maintainers will handle)
- [ ] Added Documentation (Service and Code when required)

# Screenshots

## Before
image

## After
image

### CODE REVIEW

**Comment by vinzruzell** [comment]
Pls review.

**Comment by DeeDeeG** [comment]
Thank you for submitting these translations.

The files you added look fine. Travis CI testing is showing errors in the pull requests you submitted, but I don't think the errors are caused by anything you did.

The errors all look similar to this:

```
      Timed out waiting for response to {"id":"1613a367-d27d-469a-b8f2-8c340f41f972","name":"visible","args":[2,0]}. It's possible that this happened because something took a very long time (for example a page load was slow). If so, setting the Poltergeist :timeout option to a higher value will help (see the docs for details). If increasing the timeout does not help, this is probably a bug in Poltergeist - please report it to the issue tracker. (Capybara::Poltergeist::TimeoutError)
```

Our simulated web browser that tests our webpage is having problems. We will have to try and figure out why that's happening, but I don't think it should hold up merging these files.

**Comment by DeeDeeG** [comment]
We can accept pull requests for individual files, but we usually accept all the `filename.locale.yml` files for a language in a single Pull Request.

Let us know if you prefer submitting each translation file in its own Pull Request, or if you'd be okay with waiting until they are all done, and submitting them in a single pull request.

If you want to work on the files using GitHub.com's web interface, please visit your fork and make changes there: https://github.com/vinzruzell/refugerestrooms. Then you can click the "Pull Request" button when you are ready with all the files.

pull request button github

(Sorry if you already know this information, but I prefer to over-explain rather than under-explain.)

(Note: If we accept one file at a time. we will probably merge into a `Filipino` branch here, then give that a final review before merging into `develop`.)

**Comment by vinzruzell** [comment]
Hi. I conducted another pull request ealry moring.but still failed during
the test. Hoping for a positive response for this. Thank you.

**Comment by DeeDeeG** [comment]
This looks good, except for a few minor things. Once those things are fixed, we can merge this. Thanks.

File: config/locales/fil/devise.fil.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Mga karagdagang mga pagsasalin ng salita ay makikita sa https://github.com/plataformatec/devise/wiki/I18n
+
+fil:
+  magbalangkas:
```
Comment (DeeDeeG) [inline config/locales/fil/devise.fil.yml]: `magbalangkas:` should be -> `devise:`

(We do not need to translate to the left <-- of the colon `:`)

Comment (DeeDeeG) [reply config/locales/fil/devise.fil.yml]: We only need to translate to the right --> of the colon `:`

File: config/locales/fil/devise.fil.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    mga kompermasyon:
```
Comment (DeeDeeG) [inline config/locales/fil/devise.fil.yml]: `mga kompermasyon:` should be -> `confirmations:`

File: config/locales/fil/devise.fil.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+      confirmed: "Ang iyong account ay matagumpay na nakomperma."
+      send_instructions: "Ikaw ay makakatanggap ng email na may kalakip na mga instruksiyon tungkol sa kung papaano i-komperma ang iyong account sa kakaunting minuto."
+      send_paranoid_instructions: "Kapag ang iyong email address ay umiiral sa aming database, ikaw ay makakatanggap ng email na may kasamang mga instruksiyon kung papaano i-komperma ang iyong account sa kakaunting minuto."
+    nabigo:
```
Comment (DeeDeeG) [inline config/locales/fil/devise.fil.yml]: `nabigo:` should be --> `failure:`

File: config/locales/fil/devise.fil.yml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+      confirmation_instructions:
+        subject: "Ang mga instruksiyon para sa kompermasyon"
+      reset_password_instructions:
+        subject: "I-reset ang mga instruksiyon para sa password"
+      unlock_instructions:
+        subject: "I-unlock ang mga Instruksiyon"
+    omniauth_callbacks:
+      failure: "Hindi makapagpatotoo sa iyo mula sa %{kind} dahil sa \"%{reason}\"."
+      success: "Matagumpay na napatotoo mula sa %{kind} account."
+    mga password:
```
Comment (DeeDeeG) [inline config/locales/fil/devise.fil.yml]: `mga password:` should be --> `passwords:`

File: config/locales/fil/devise.fil.yml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+      no_token: "Hindi ka maka-akses sa pahinang ito kung wala kang password reset mula sa iyong email. Kapag gusto mong baguhin ang password gamit ang email, dapat siguraduhin na gamitin ang buong URL na ibinigay."
+      send_instructions: "Ikaw ay makatanggap ng email na may kalakip na mga tungkol sa kung papaano ma-reset ang iyong password sa iilang minuto."
+      send_paranoid_instructions: "Kapag ang iyung email ay nasa database na namin, ikaw ay makakatanggap ng recovery link sa iyong email address your email address sa iilang minuto."
+      updated: "Ang password mo ay matagumpay na napalitan.Ikaw ngayon ay naka-signed in na."
+      updated_not_active: "Ang iyong password ay matagumpay nang napalitan."
+    mga pagpaparehistro:
```
Comment (DeeDeeG) [inline config/locales/fil/devise.fil.yml]: `mga pagpaparehistro:` should be --> `registrations:`

File: config/locales/fil/devise.fil.yml [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+      destroyed: "Paalam! Ang iyong account ay matagumpay na nakansela. Ninanais naming makita kayong muli."
+      signed_up: "Maligayang pagdating! You have signed up successfully."
+      signed_up_but_inactive: "Ikaw ay matagumpay na naka-signed up. Gayunpaman, hindi pa namin kayo mapasign in dahil ang iyong  account ay hindi pa aktibo."
+      signed_up_but_locked: "Ikaw ay matagumpay na naka-signed up. Gayunpaman,, hindi pa namin kayo mapasign in dahil ang iyong  account ay nakalock."
+      signed_up_but_unconfirmed: "Ang mensahi na may kompermasyon sa link ay naipadala na sa iyong email address. Kami ay nakikiusap na buksan ang link para mapagana ang iyong account."
+      update_needs_confirmation: "Ang iyong account ay matagumpay nang na-updeyt, pero kinakailangan pa naming i-verify ang iyong bagong email address. Kami ay nakiki-usap na tingnan ang iyong email at mag-klik sa confirm link para ma-finalize ang pagkomperma sa iyong bagong email address."
+      updated: "Matagumpay mong na-update ang iyong account."
+    mga sesyon:
```
Comment (DeeDeeG) [inline config/locales/fil/devise.fil.yml]: `mga sesyon:` should be --> `sessions:`

File: config/locales/fil/devise.fil.yml [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+      signed_in: "Matagumpay kang naka-signed in."
+      signed_out: "Matagumpay kang naka-signed out."
+    mga unlock:
```
Comment (DeeDeeG) [inline config/locales/fil/devise.fil.yml]: `mga unlock:` should be --> `unlocks:`

File: config/locales/fil/devise.fil.yml [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+      send_instructions: "Ikaw ay makakatanggap ng email namay kalakip na mga instruksiyon tungkol sa kung papaano ma-unlock ang iyong account sa iilang minuto."
+      send_paranoid_instructions: "Kapag ang iyong account ay umiiral, ikaw ay makakatanggap ng email namay kalakip na instruksiyon tungkol sa kung papaano ito ma-unlock sa iilang minuto."
+      unlocked: "Ang iyong account ay matagumpay nang na-unlock. Mag-sign in para makapagpatuloy."
+  mga error:
```
Comment (DeeDeeG) [inline config/locales/fil/devise.fil.yml]: `mga error:` should be --> `errors:`

File: config/locales/fil/devise.fil.yml [hunk 9]
```diff
@@ -0,0 +1,59 @@
...
+    mga mensahe:
```
Comment (DeeDeeG) [inline config/locales/fil/devise.fil.yml]: `mga mensahe:` should be --> `messages:`

**Comment by vinzruzell** [comment]
HI @DeeDeeG I have already change the text back to its original text. Thank you.

**Comment by vinzruzell** [comment]
HI @DeeDeeG  Pls let me know if still have some problem. Thank you.

**Comment by DeeDeeG** [comment]
> HI @DeeDeeG I have already change the text back to its original text. Thank you.

Alright, this is ready to merge now, thank you.

> HI @DeeDeeG Pls let me know if still have some problem. Thank you.

No problem. I/the team are not available to review things 24 hours a day, so sometimes there will be a delay for reviewing your changes.

**Comment by vinzruzell** [comment]
Thank you for that :)

**Pull request merged**